OpenRA.Mods.Common/Widgets/Logic/Ingame/ObserverStatsLogic.cs

```diff
@@ -113,7 +143,7 @@ public ObserverStatsLogic(World world, ModData modData, WorldRenderer worldRende
 			{
 				return new StatsDropDownOption
 				{
-					Title = title,
+					Title = modData.Translation.GetString(title),
```

Comment:
You should also apply this in line 152, otherwise not everything gets translated